### PR TITLE
Using filename for jira issue find

### PR DESCRIPTION
**Pull request by Legion18z**

### CODE REVIEW

File: vedro_jira_failed_reporter/_failed_reporter_plugin.py [hunk 1]
```diff
@@ -183,10 +185,9 @@ def on_scenario_failed(self, event: Union[ScenarioPassedEvent, ScenarioFailedEve
             self._reporting_language.ISSUE_CREATED.format(jira_server=self._jira_server, issue_key=result_issue.key)
         )
 
-        traceback = event.scenario_result._step_results[-1].exc_info.traceback
```
Comment (ko10ok) [inline vedro_jira_failed_reporter/_failed_reporter_plugin.py:186]: we don't need this section (issues linking) anymore i suppose?

if we make one issue per file we don't have any separate issues related with this file.

Comment (ko10ok) [inline vedro_jira_failed_reporter/_failed_reporter_plugin.py:186]: i suppose we don't need whole issues linking section anymore.

if we will make single issue per file we won't get any separate related issues. So no need to search and link any at all.

Comment (Legion18z) [reply vedro_jira_failed_reporter/_failed_reporter_plugin.py:186]: Code for creating links was removed.

**Comment by ko10ok** [comment]
LGTM! Thx!

**Pull request merged**